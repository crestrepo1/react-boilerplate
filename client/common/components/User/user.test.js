import React from 'react';
import { shallow } from 'enzyme';
import UsersView from './index.js';

describe('UserView component', () => {
    it('should render correctly', () => {
        const component = shallow(<UsersView />);
        expect(component).toMatchSnapshot();
    });
});
