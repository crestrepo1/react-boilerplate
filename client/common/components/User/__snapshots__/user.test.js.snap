// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserView component should render correctly 1`] = `
<div>
  <button
    onClick={[Function]}
  >
    fetch User
  </button>
</div>
`;
